sdk/keyvault/azure-security-keyvault-common/inc/azure/keyvault/common/version.hpp

```diff
@@ -17,22 +17,22 @@
 #define AZURE_SECURITY_KEYVAULT_COMMON_VERSION_PATCH 0
 #define AZURE_SECURITY_KEYVAULT_COMMON_VERSION_PRERELEASE "beta.1"
 
-namespace Azure { namespace Security { namespace KeyVault { namespace Common {
+namespace Azure { namespace Security { namespace KeyVault { namespace Common { namespace _detail {
```

Comment:
so there's actually a Common namespace :)

Reply:
I wonder if we should rename this namespace to `KeyVaultCommonVersion` instead of `KeyVault::Common::_detail`

Reply:
Separate discussion from this PR